drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -76,7 +76,7 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
   # Non-configurable forms. These are submitted using custom mutations.
   STATIC_FORM_ROLES = [
     :FORM_RULE,
-    :AUTO_EXIT_CONFIG,
+    :PROJECT_CONFIG,
```

Comment:
@martha I think because removed the `AUTO_EXIT_CONFIG` role here which drops it from the graphql API. Lets leave it and we can remove it later once the migrations have been run

Reply:
Good thinking. Thanks!